rollup.build.js

```diff
@@ -80,6 +80,46 @@ export default [].concat.apply(
                     src: `./dev-lib/bundle-${variant}.d.ts`,
                     dest: ".",
                     rename: umdFileWithoutExt + ".d.ts"
+                  },
+                  {
+                    src: "./dev-lib/bundle-esm-index.d.ts",
+                    dest: ".",
+                    rename: `${variant}/index.d.ts`
```

Comment:
Just a suggestion, but maybe you can right this in a cleaner way. Something like:
```javascript
const jsNames = [
'index.js',
'esm/index.d.ts',
'esm/index.js',
'umd/index.d.ts',
'umd/index.js',
]

const cssNames = [
'index.d.ts',
'index.js'
]

const jsBuilds = jsNames.map((jsName) => ({
  src: "./dev-lib/bundle-vis-network-index.d.ts",
  dest: ".",
  rename: `${variant}/${jsName}`
}))

const cssBuilds = cssNames.map((cssName) => ({
  src: "./dev-lib/bundle-vis-network-index.d.ts",
  dest: ".",
  rename: `styles/${cssName}`
}))
```

and then add them to the builds?

Reply:
I though about such solution when I was writing this but I'm not sure it's clearer. The way it is now it basically says copy A to B, A to C, D to E, D to F, etc. Pretty straightforward and easy to understand albeit verbose. With your solution there would be a bunch of arrays and mapping over them and it would actually take about the same number of lines of code (your code won't work because index.js is different from esm/index.js, it would require separate array and map) to achieve the same in quite cryptic and hard to follow way.

Reply:
Ok. Sounds fair to me.